examples/okta_group_rule/basic_updated.tf

```diff
@@ -7,5 +7,5 @@ resource "okta_group_rule" "test" {
   status            = "ACTIVE"
   group_assignments = ["${okta_group.test_other.id}"]
   expression_type   = "urn:okta:expression:1.0"
-  expression_value  = "String.startsWith(user.articulateId,\"auth0|\")"
+  expression_value  = "String.startsWith(String.toLowerCase(user.articulateId,\"auth0|\"))"
```

Comment:
I think if you fix that expression, then it will be good to merge! (notice it previously had two args, this update nested those two args in a new function call)
```
The API returned an error: Api validation failed: validateRule. Causes: errorSummary: Parameter mismatch - expected class com.saasure.platform.cvd.expression.functions.StringFunctions.startsWith but found 2 args, errorSummary: Parameter mismatch - expected class com.saasure.platform.cvd.expression.functions.StringFunctions.toLowerCase but found 1 args, errorSummary: conditions: Invalid expression
```

Reply:
Absolutely. Sorry, fat fingered a function that clearly takes a single argument. )